pom.xml

```diff
@@ -82,5 +82,11 @@
 		</profile>
 	</profiles>
 	<dependencies>
-	</dependencies>  
```

Comment:
Is there a white space difference here? Just wondering why `</dependencies>` thinks it's changed.

Reply:
I am not sure...